src/listener.c

```diff
@@ -264,6 +264,18 @@ accept_listener_protocol(struct Listener *listener, char *protocol) {
     return 1;
 }
 
+#ifdef SO_REUSEPORT
+int
+accept_listener_reuseport(struct Listener *listener, char *reuseport) {
+    if (strncasecmp(reuseport, "yes", strlen(reuseport)) == 0)
```

Comment:
See above comments. I would suggest using ifdef around behavior rather than function definition. We could pull `parse_boolean_argument()` into the config.c so we can support `on`, `yes`, `true`, etc.